tekton/README.md

```diff
@@ -60,28 +60,28 @@ So for example, we might want to run one or more test releases under the name 't
 - Go to https://console.cloud.google.com/storage/browser/tekton-releases/dashboard and click 'Create folder'. Create the folder Buckets/tekton-releases/dashboard/test-release.
 - Modify the tekton-bucket-dashboard PipelineResource:
 
-```yaml
-apiVersion: tekton.dev/v1alpha1
-kind: PipelineResource
-metadata:
-  name: tekton-bucket-dashboard
-spec:
-  type: storage
-  params:
-   - name: type
-     value: gcs
-   - name: location
-     value: gs://tekton-releases/dashboard/test-release # If you're testing use your bucket name here instead of test-release
-   - name: dir
-     value: "y"
-```
+  ```yaml
+  apiVersion: tekton.dev/v1alpha1
+  kind: PipelineResource
+  metadata:
+    name: tekton-bucket-dashboard
+  spec:
+    type: storage
+    params:
+     - name: type
+       value: gcs
+     - name: location
+       value: gs://tekton-releases/dashboard/test-release # If you're testing use your bucket name here instead of test-release
+     - name: dir
+       value: "y"
+  ```
 
 - Apply your changes
 
-```bash
-PIPELINE_NAMESPACE=tekton-pipelines
-kubectl apply -f tekton -n $PIPELINE_NAMESPACE
-```
+  ```bash
+  PIPELINE_NAMESPACE=tekton-pipelines
```

Comment:
Shouldn't all these refs be

```
PIPELINE_NAMESPACE=dashboard-release-pipeline
```

? That's what we refer to in the tkn command for example

Reply:
`dashboard-release-pipeline` is only used in the OpenShift instructions, the first set all reference `tekton-pipelines`

Reply:
Good point